examples/simple/main.py

```diff
@@ -2,11 +2,14 @@
 import fairing
 import tensorflow as tf
 
-class SimpleModel():
-    def train(self):
-        hostname = tf.constant(os.environ['HOSTNAME'])
-        sess = tf.Session()
-        print('Hostname: ', sess.run(hostname).decode('utf-8'))
+fairing.config.set_builder('docker', push=False)
+fairing.config.set_deployer('job')
 
-fairing.config.set_model(SimpleModel())
-fairing.config.run()
\ No newline at end of file
+def train():
+    hostname = tf.constant(os.environ['HOSTNAME'])
+    sess = tf.Session()
+    print('Hostname: ', sess.run(hostname).decode('utf-8'))
+
+if __name__ == '__main__':
+    train = fairing.config.fn(train)
```

Comment:
giving the new function the same name ("train") is confusing. Probably something like remote_train = fairing.config.fn(train); remote_train() is less confusing.

Reply:
Sure. What I imagined was that the user could add/remove this line and keep the rest of their code the same to do local/remote training.